package.json

```diff
@@ -44,18 +44,19 @@
     "@babel/core": "^7.18.5",
     "@emotion/react": "11.10.4",
     "@emotion/styled": "11.10.4",
+    "@fontsource/material-icons": "5.1.1",
```

Comment:
Why do we need this?

Reply:
this is part of the configuration process to integrate Mui with storybook. I tried to keep the old way were the fonts were loaded in the `manage-head.html` but it was not being applied.

![Screenshot 2025-01-13 at 15 02 35](https://github.com/user-attachments/assets/93e95e30-648d-483e-a058-8da7333d8e0a)

Reply:
@marcosh-ramos I believe this is different in v5: https://v5.mui.com/material-ui/getting-started/installation/#roboto-font

Reply:
I removed the material icons package https://github.com/storyblok/mui/pull/46#:~:text=View%20deployment-,chore%3A%20remove%20%40fontsource/material%2Dicons%20package,-3e5663a